README.md

```diff
@@ -47,48 +51,71 @@ end
     be ensured by calling `Circuitry.flush`.
-  * `:batch`: Stores the request in memory to be submitted later.  Batched
+  * `:batch`: Stores the request in memory to be submitted later. Batched
     requests must be manually sent by calling `Circuitry.flush`.
 * `subscribe_async_strategy`: One of `:fork` or `:thread` that determines how
-  asynchronous subscribe requests are processed.  *(optional, default: `:fork`)*
+  asynchronous subscribe requests are processed. *(optional, default: `:fork`)*
   * `:fork`: Forks a detached child process that immediately begins querying the
     queue.
   * `:thread`: Creates a new thread that immediately sends begins querying the
     queue.
-* `on_thread_exit`: An object that responds to `call`.  This is useful for
+* `on_thread_exit`: An object that responds to `call`. This is useful for
   managing shared resources such as database connections that require closing.
-  It is only called when implementing the `:thread` async strategy.  *(optional,
+  It is only called when implementing the `:thread` async strategy. *(optional,
   default: `nil`)*
-* `on_fork_exit`: An object that responds to `call`.  This is useful for
+* `on_fork_exit`: An object that responds to `call`. This is useful for
   managing shared resources such as database connections that require closing,
-  It is only called when implementing the `:fork` async strategy.  *(optional,
+  It is only called when implementing the `:fork` async strategy. *(optional,
   default: `nil`)*
+* `subscriber_queue_name`: The name of the SQS queue that your subscriber application
+  will listen to. This queue will be created or configured during `rake circuitry:setup`
+  *(optional, default: `nil`)*
+* `subscriber_dead_letter_queue_name`: The name of the SQS dead letter queue that will be
+  used after all retries fail. This queue will be created and configured during `rake
+  circuitry:setup` *(optional, default: `<subscriber_queue_name>-failures`)*
+* `publisher_topic_names`: An array of topic names that your publishing application will
+  publish on. This configuration is only used during provisioning via `rake circuitry:setup`
+
+### Provisioning
+
+You can automatically provision SQS queues, SNS topics, and the subscriptions between them using two methods: the circuitry CLI or the `rake circuitry:setup` task. The rake task will provision the subscriber queue and publishing topics that are configured within your application.
+
+```ruby
+Circuitry.config do |c|
+  c.subscriber_queue_name = 'myapp-production-events'
+  c.publisher_topic_names = ['theirapp-production-stuff-created', 'theirapp-production-stuff-deleted']
+end
+```
+
+When provisioning, a dead letter queue is also created using the name "<queue_name>-failures" and a redrive policy of 8 retries to that dead letter queue is configured. You can customize the dead letter queue name in your configuration.
+
+Run `ruby bin/circuitry help provision` for help using CLI provisioning.
```

Comment:
I believe this can just be `circuitry help provision` as bundler will add circuitry to the ruby bin folder, which rvm/rbenv includes in the PATH.